src/cascadia/TerminalSettingsEditor/MainPage.cpp

```diff
@@ -624,6 +622,7 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
         const auto newTag = newSelectedItem.as<MUX::Controls::NavigationViewItem>().Tag();
         if (const auto profileViewModel = newTag.try_as<ProfileViewModel>())
         {
+            profileViewModel->FocusDeleteButton(true);

```

Comment:
Do we ever clear this back to false? Or will that profile always remember that it was supposed to focus the delete button?

Reply:
Yep we do! In `Profiles_Base.cpp`, after we focus the delete button